src/backend/distributed/executor/multi_utility.c

```diff
@@ -572,6 +572,25 @@ ProcessAlterTableStmt(AlterTableStmt *alterTableStatement, const char *alterTabl
 static void
 ErrorIfUnsupportedIndexStmt(IndexStmt *createIndexStatement)
 {
+	Oid namespaceId;
+	Oid indexRelationId;
+	char* indexRelationName = createIndexStatement->idxname;
+
+	if (indexRelationName == NULL)
+	{
+		ereport(ERROR, (errcode(ERRCODE_DUPLICATE_TABLE),
```

Comment:
I was pretty sure I wrote about this part but can not see it now.
```ERRCODE_DUPLICATE_TABLE``` and having a no name for index is irrelevant.

Also
we usually use 'xxxxx not supported' like error messages. you could state something like 'creating an index without a name is not supported for distributed tables'.